lld/ELF/AArch64ErrataFix.cpp

```diff
@@ -461,12 +461,11 @@ void AArch64Err843419Patcher::init() {
     llvm::stable_sort(mapSyms, [](const Defined *a, const Defined *b) {
       return a->value < b->value;
     });
-    mapSyms.erase(
-        std::unique(mapSyms.begin(), mapSyms.end(),
```

Comment:
I believe this was output from a previous version of clang-format. Personally I'd avoid such a change, but if a special rule makes things harder, I think it would be fine to format this.